src/stores/PolygonDrawingStore/PolygonDrawingStore.types.ts

```diff
@@ -6,6 +6,17 @@ export type PolygonFeature = FeatureOf<Polygon>;
 
 export type Point2D = [number, number];
 
+export type cellMetrics = {
+  type: string; // 'protein' | 'RNA'
```

Comment:
If the type can only be 'protein' or 'RNA', can we create a separate `CellMetricsType` and use it here, or will there be any issues?

```
export type CellMetrics = {
  type: CellMetricsType;
  ...
};

export type CellMetricsType = 'protein' | 'RNA';
```